src/main/java/chess/controller/api/ApiController.java

```diff
@@ -0,0 +1,78 @@
+package chess.controller.api;
+
+import chess.dto.MovePositionReq;
+import chess.dto.PasswordReq;
+import chess.dto.RoomAllRes;
+import chess.dto.RoomCreateReq;
+import chess.dto.StatusRes;
+import chess.entity.Room;
+import chess.service.ChessService;
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+
+    private ChessService chessServiceV2;
+
+    public ApiController(ChessService chessServiceV2) {
+        this.chessServiceV2 = chessServiceV2;
+    }
+
+    @GetMapping("/room/all")
```

Comment:
```suggestion
    @GetMapping("/rooms")
```
이렇게 해도 충분하지 않을까 싶어요 :)